<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThermodynamicsInteraction</class>
 <widget class="QWidget" name="ThermodynamicsInteraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxInteractions">
     <property name="title">
      <string>List of interactions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableViewInteractions"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxGeneral">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSurfaceTensionValue">
        <property name="text">
         <string>Surface tension value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxSurfaceTensionValue"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditSurfaceTensionValue"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelUnitSurfaceTensionValue">
        <property name="minimumSize">
         <size>
          <width>37</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>J/m&lt;sup&gt;2&lt;/sup&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButtonSurfaceTension">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/icons/22x22/edit-find-replace.png</normaloff>
          :/new/prefix1/icons/22x22/edit-find-replace.png
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
